libraries/AP_Baro/AP_Baro_Backend.cpp

```diff
@@ -86,11 +86,10 @@ bool AP_Baro_Backend::pressure_ok(float press)
     } else {
         const float d = fabsf(_mean_pressure - press) / (_mean_pressure + press);  // diff divide by mean value in percent ( with the * 200.0f on later line)
         float koeff = FILTER_KOEF;
-
```

Comment:
i'd like to see this as a new object in libraries/Filter, with unit test

Reply:
sure I will do my best :)